include/z64audio.h

```diff
@@ -539,10 +540,8 @@ typedef struct Note {
     /* 0x00 */ AudioListItem listItem;
     /* 0x10 */ NoteSynthesisState synthesisState;
     /* 0x30 */ NotePlaybackState playbackState;
-    /* 0x90 */ Portamento portamento;
-    /* 0x9C */ VibratoState vibratoState;
     /* 0xB8 */ char unk_B8[0x4];
-    /* 0xBC */ u32 unk_BC;
+    /* 0xBC */ u32 startSamplePos; // initial position/index to start processing s16 samples. Always set to 0
```

Comment:
This is more of a general note and it might just be me, but I'm not a fan of comments like "Always set to 0" since they essentially invalidate the point of the variable and don't provide any info you couldn't get on your own with a simple search. It's also not the kind of comment you would have in a real codebase, even if it was true